env.example

```diff
@@ -2,7 +2,10 @@
 // This is only a .env example
 // Do not change this file, adding your GITHUB TOKEN here!
 // Use cp or mv (like this: cp env.example .env), then edit .env with your GITHUB TOKEN.
-// IMPORTANT: Don't forget to add to update your .gitignore with .env (to avoid public share your key!)
+// IMPORTANT: Don't forget to add to update your .gitignore with .env (to avoid making your key public!)
 
 REACT_APP_GITHUB_TOKEN = "YOUR GITHUB TOKEN HERE"
 GITHUB_USERNAME = "YOUR GITHUB USERNAME HERE"
+// Set to true to fetch profile data from GitHub (remember to remove all components relying on GitHub data if set to false)
+// Ensure data type is string
```

Comment:
this comment doesn't make sense to me since this is a `.env` file where things should be always as a string.

Reply:
Fair point but I wanted to clear confusion for new folks about not thinking of it as a boolean and remove `""`. 
I'll remove the comment.